website/addons/dataverse/settings/defaults.py

```diff
@@ -1,5 +1,6 @@
 DEFAULT_HOSTS = [
     'dataverse.harvard.edu',            # Harvard PRODUCTION server
+    'dataverse.lib.virginia.edu',       # UVA PRODUCTION server
     'dataverse-demo.iq.harvard.edu',    # Harvard DEMO server
```

Comment:
That was actually resolved in another [PR](https://github.com/CenterForOpenScience/osf.io/pull/4979/files#diff-dc253141a64ffc81e0032ed7676e8b06L3) recently merged into `develop`. This just hasn't been updated since then. Thanks for letting us know, though :+1: